source/content/site-dashboard.md

```diff
@@ -27,6 +27,7 @@ On the top left of the page, you will see the following information:
 - If applicable, a yellow **Upgrade** button, that will allow you to [upgrade the site](/guides/account-mgmt/plans/site-plans).
 - If applicable, the name of the Professional Workspace containing this site.
 
+<br />
```

Comment:
I think there needs to be more space above a new paragraph following a list so I threw in the `<br />` but I can do a separate PR to address that in the template styles instead of this approach 
<img width="1695" alt="Screenshot 2024-12-18 at 9 28 40 AM" src="https://github.com/user-attachments/assets/59a22302-8b1d-4033-a372-5036e554d0f0" />